### DISCUSSION TITLE
I cannot transcribe a local file of PCM data.

### DISCUSSION THREAD

**Discussion by couldbejake** [deepgram/community#429] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

I am attempting to transcribe the local file `48k_peter_jackson.pcm`.
The deepgram node.js client returns no transcription.

The audio file contains 40 seconds of spoken text in a New Zealand accent. The audio file transcribes perfectly on other services such as Google SST, Whisper and Rev SST. The audio is sampled at 48khz, yet I see no option to set the sample rate of prerecorded audio on deepgram.

The response from the API shows:

```
{
  metadata: {
    transaction_key: 'deprecated',
    request_id: '583ff5e3-defb-4a1d-b6ef-f99c7a4bd5ba',
    sha256: 'de49bb76eb8b0aac2797f89f6d370e3b53b81c6d70a0e2b4392ca8431517a1a8',
    created: '2023-11-08T20:23:07.552Z',
    duration: 2.8628125,
    channels: 1,
    models: [ '96a295ec-6336-43d5-b1cb-1e48b5e6d9a4' ],
    model_info: {
      '96a295ec-6336-43d5-b1cb-1e48b5e6d9a4': { name: 'general', version: '2023-02-22.3', arch: 'base' }
    }
  },
  results: {
    channels: [
      {
        alternatives: [ { transcript: '', confidence: 0, words: [] } ]
      }
    ]
  }
}
```

No transcription is given.

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

583ff5e3-defb-4a1d-b6ef-f99c7a4bd5ba

### If possible, please attach your code or paste it into the text box.

```
const fs = require('fs')
const { Deepgram } = require('@deepgram/sdk');

// The API key we created in step 3
const deepgramApiKey = '<API KEY>';

// Initializes the Deepgram SDK
const deepgram = new Deepgram(deepgramApiKey);

console.log('Requesting transcript...')
console.log('Your file may take up to a couple minutes to process.')
console.log('While you wait, did you know that Deepgram accepts over 40 audio file formats? Even MP4s.')
console.log('To learn more about customizing your transcripts check out developers.deepgram.com.')

deepgram.transcription.preRecorded(
    {
        buffer: fs.readFileSync("./48k_peter_jackson.pcm"),
        mimetype: "audio/linear16",
    },
  {smart_format: true, model: 'nova', language: 'en-US', sample_rate: 48000 },
)
.then((transcription) => {
  console.dir(transcription, {depth: null});
})
.catch((err) => {
  console.log(err);
});
```

**Comment by DamienDeepgram** [comment, marked as answer]
For Batch Transcription you will need to convert the file to a non-raw format 

Here is an example how you can convert the file from raw to wav

> ffmpeg -ar 16000 -f s16le -ac 1 -i test.pcm test.wav

**Comment by couldbejake** [comment]
> For Batch Transcription you will need to convert the file to a non-raw format 
> 
> 
> 
> Here is an example how you can convert the file from raw to wav
> 
> 
> 
> > ffmpeg -ar 16000 -f s16le -ac 1 -i test.pcm test.wav

Thank you, works perfectly
